src/test/java/it/pagopa/pn/delivery/middleware/notificationdao/PaNotificationLimitDaoDynamoTest.java

```diff
@@ -0,0 +1,101 @@
+package it.pagopa.pn.delivery.middleware.notificationdao;
+
+import it.pagopa.pn.delivery.PnDeliveryConfigs;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
```

Comment:
Bene il test junit specifico, potrebbe secondo me essere utile un test IT con localStack come fatto per le altre classi che verifichi esattamente i nostri casi. in modo da validarli con test specifici